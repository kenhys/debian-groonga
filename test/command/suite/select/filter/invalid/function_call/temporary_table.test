#@include fixture/geo/taiyaki/init.grn

select Shops \
  --output_columns 'name' \
  --limit 0 \
  --filter 'geo_in_rectangle(location, "90.0x180.0", "90.0x180.0") && \
            (area == "area0005" || area == "area0014") && \
            (tags @ "天然" || tags @ "和菓子")'


