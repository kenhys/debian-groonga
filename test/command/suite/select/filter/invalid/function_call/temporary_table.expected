select Shops   --output_columns 'name'   --limit 0   --filter 'geo_in_rectangle(location, "90.0x180.0", "90.0x180.0") &&             (area == "area0005" || area == "area0014") &&             (tags @ "天然" || tags @ "和菓子")'
[
  [
    [
      -22,
      0.0,
      0.0
    ],
    "geo_in_rectangle(): failed to cast to WGS84GeoPoint: <90.0x180.0>"
  ]
]
#|e| geo_in_rectangle(): failed to cast to WGS84GeoPoint: <90.0x180.0>
